shopinvader_sale_configurator_option/models/product_product.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Akretion (https://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    shopinvader_price = fields.Serialized(
+        compute="_compute_shopinvader_price", string="Shopinvader Price"
```

Comment:
This field label seems to conflict with the `price` field https://github.com/shopinvader/odoo-shopinvader/blob/14.0/shopinvader/models/shopinvader_variant.py#L60

(Getting warning: `Two fields (shopinvader_price, price) of [] have the same label: Shopinvader Price.`)